src/components/DropZone/style/DropZone.style.ts

```diff
@@ -0,0 +1,147 @@
+import styled, { css } from 'styled-components';
+import { Text } from '../../Typography/Text';
+import { Icon } from '../../Icon';
+import { Title } from '../../Typography/Title';
+import { Type, Size } from 'src/types';
+
+interface StyledDropZoneProps {
+    size: Size;
+}
+
+interface StyledDropZoneIconProps {
+    type: Type;
+    size: Size;
+}
+
+interface StyledUploadingTitle {
+    size: Size;
+}
+
+export const StyledDropZoneText = styled(Text)``;
+
+export const StyledSmallText = styled(Text)`
+    font-size: 10px;
+`;
+
+export const StyledDropZone = styled.div<StyledDropZoneProps>`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    box-sizing: paddind-box;
+    padding: ${(props) => props.theme.padding.lg};
+    margin: ${(props) => props.theme.margin.lg};
+    border: 1px dashed ${(props) => props.theme.color.border};
+
+    &:hover {
+        border: 1px dashed ${(props) => props.theme.color.primary.alpha};
```

Comment:
je potřeba zde nastavovat opět stejné hodnoty, co jsi nastavila o tři řádky výš? Nemohli bychom v hoveru nastavit jen barvu borderu? :)
Je to i z toho důvodu, že kdybys náhodou chtěla změnit typ nebo šířku borderu tak to musíš upravovat na dvou místech.